msg/RadarDetection.msg

```diff
@@ -0,0 +1,11 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+# Radar detections are the discrete clusterings of points
+# Detections which are filtered into moving objects are referred to as Tracks
+
+                                            # NB: The z component of these fields is ignored for 2D tracking.
+geometry_msgs/Point position                # x, y, z coordinates of the centroid of the object being tracked.
+geometry_msgs/Vector3 velocity              # The velocity of the object in each spacial dimension.
+geometry_msgs/Vector3 size                  # The RCS or average deviation (m) in the x, y, z, dimensions
```

Comment:
RCS is very different than the clustering of the points. They shouldn't be conflated.

It would probably make sense to provide some sort of aggregate of the intensity of the detections as captured by the average RCS for the points being aggregated. 

In the tracking message you picked a bounding box these should be consistent. #3

Reply:
Agreed, I dont think any message API needs to be updated, but the description should be

Reply:
> For Position - did any of the Poses use the orientation information? 
The https://github.com/Project-MANAS/ars_40X driver does appear to populate the orientation information but I don't really understand what is going on in the code.

> For velocity - Did any of the Twists use the angular velocity entries? If not, then Vector3 is the right choice.
The information for twist appears to come from an odometry message which the driver subscribes to